tests/test_globalization.py

```diff
@@ -3,91 +3,148 @@
 import socket
 from utility import random_str, decode
 from mssqltestutils import create_mssql_cli_client, shutdown
+from mssqlcli.mssqlcompleter import MssqlCompleter
+import mssqlcli.completion_refresher as completion_refresher
+from prompt_toolkit.document import Document
+
+
+"""
+All tests require a live connection to test server.
+Make modifications to mssqlutils.create_mssql_cli_client()
+to use a different server and database.
+Please Note: These tests cannot be run offline.
+"""
 
 
-# All tests require a live connection to test server.
-# Make modifications to mssqlutils.create_mssql_cli_client() to use a different server and database.
-# Please Note: These tests cannot be run offline.
 class GlobalizationResultSetTests(unittest.TestCase):
     
     def test_charset_double(self):
-        charset_list = self.get_charset_double() 
-        self.run_charset_validation(charset_list)
+        charset = GB18030()
+        characters_list = charset.\
```

Comment:
nit: rather than use explicit line continuations via the backslash, can you break on a { or a ( which are implicit line continuations?

Reply:
fixed as you requested. I am very sure it will complain on the implicit way when the project uses pep8